Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -76,8 +77,8 @@ bool AVIDump::Start(int w, int h)
 {
   s_pix_fmt = AV_PIX_FMT_RGBA;
 
-  s_width = w;
-  s_height = h;
+  s_width = g_Config.iDumpWidth ? g_Config.iDumpWidth : w;
```

Comment:
@RisingFog Then unset it in the ini? We could also postpone some of this UI stuff, since the default behavior (with the sizes unset) matches the old behavior.

@ligfx You might want to lower the resolution to save space? I agree it's not the greatest solution to the variable frame problem, but I don't know of a better alternative. I think this might even be the desired behavior for some frame resolution changes (like cutscenes), although we might want to do something slightly different to avoid changing the aspect ratio.